_rules/video-only-alternative-for-visual-c3232f.md

```diff
@@ -75,7 +77,7 @@ See [Video Only Element Has Description Track: accessibility support](https://ac
 
 #### Passed Example 1
```

Comment:
I'm considering whether we should take the opportunity to reorder the examples in "most common to least common" order. I feel that having a video as an alternative for text is like super rare and that transcript or video is more common.

Reply:
That makes sense. Updating accordingly.